src/public/orders/sales-order.ts

```diff
@@ -23,7 +27,7 @@ export interface SalesOrder extends SalesOrderIdentifierPOJO {
   /**
    * Indicates how the customer has paid for the order
    */
-  paymentMethod?: PaymentMethod;
+  paymentMethod?: string;
```

Comment:
@justin-shipstation I'm curious about this, what values are we getting that we can't map? Is it similar to the Order Source Status we talked about the other day?

Reply:
@jeversmann-shipstation might have more context on this one.

Reply:
This is for compatibility with Custom Store, which didn't restrict the values for payment method: https://help.shipstation.com/hc/en-us/articles/360025856192-Custom-Store-Development-Guide#UUID-1bb7ab47-6cf4-c6cd-0e5b-dec61e91201b

We can recommend in the docs that developers on new integrations continue to map their payment methods to the enum, but we need to be able to allow arbitrary strings.